etc/scripts/synclic.py

```diff
@@ -45,11 +45,13 @@
 
 from commoncode import fetch
 from commoncode import fileutils
+from commoncode import compat
```

Comment:
Merge this commit into https://github.com/nexB/scancode-toolkit/pull/1534/commits/de83acebea61f5d7608bf50060d407aedf22322d .